modules/assim.sequential/R/sda_plotting.R

```diff
@@ -769,7 +769,7 @@ post.analysis.multisite.ggplot <- function(settings, t, obs.times, obs.mean, obs
   
 
   suppressMessages({
-      aoi_boundary_HARV <- sf::st_read(system.file("extdata", "eco-regionl2.json", package = "PEcAnAssimSequential"))
+    aoi_boundary_HARV <- sf::st_read(system.file("extdata", "eco-regionl2.json", package = "PEcAnAssimSequential"))
```

Comment:
Is this really just Harvard or just using this as a general name for alll locations?

Reply:
So I am not sure why this is showing up as changed I did not edit any of the sda_plotting functions except to change the package name. @DongchenZ would know better than me he has spent some time improving the plotting functions.